src/modules/auth/dto/login.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
```

Comment:
I think that you should move this file into this folder, the correct path should be:

`/src/modules/auth/models/dto/login.dto.ts`

just add "models" folder

Reply:
done!